components/content/Callout.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div class="bg-surface-secondary rounded-lg">
+    <div class="flex p-5 space-x-5">
+      <information-circle class="icon-md text-static-black" />
+      <div class="flex flex-col align space-y-1 text-sm leading-5">
+        <div
```

Comment:
You probably stole this from Tailwind but could this be a `<p>` element instead of a `<div>`?